app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,6 @@
+class SessionsController < ApplicationController
```

Comment:
At first glance it looks like 3-way oauth, and maybe just a stop gap to get the initial authentication working.

But, I'd suggest two things here:

1. the class name should reflect oauth somehow
2. the page should auto-redirect and not ask the user to close the browser

@Fryguy, @matthewd, @dclarizio ... what are your thoughts on this?